### DISCUSSION TITLE
Effect of reaction on Smoke temperature

### DISCUSSION THREAD

**Discussion by RNCFD** [firemodels/fds#13314] [General]
Hello,

Is there someone who can help me understand why with a 100% similar simulation setup (car park with burning cars in a certain sequence), but only different reaction, I get different smoke temperatures?
 
The reaction and fuels are:
Fuel 1:
&SPEC ID='SFPE POLYURETHANE_GM37_fuel', FORMULA='C1.0H1.2O0.2N0.08'/
&REAC ID='Car Fire', FYI='SFPE Handbook, 5th Edition, Tables A.38 and A.39',
FUEL='SFPE POLYURETHANE_GM37_fuel',
CO_YIELD=8.0E-3, SOOT_YIELD=0.06, HEAT_OF_COMBUSTION=2.0E+4, RADIATIVE_FRACTION=0.35/
Fuel 2: 
&REAC ID='FDS6 N-HEPTANE', FYI='FDS6 Predefined',
FUEL='N-HEPTANE',
CO_YIELD=8.0E-3, SOOT_YIELD=0.06, HEAT_OF_COMBUSTION=2.0E+4, RADIATIVE_FRACTION=0.35/

I measured the temperature in the fire under the ceiling, see results for 1 location:
![Picture1](https://github.com/user-attachments/assets/2bfdbc24-5dd1-480c-ba08-484596c00063)

As demonstrated above the tmperatures of the Polyurethane reaction are higher. I would expect Polyurethane to burn colder than Heptane, but this is not the case. Could you explain why this happens?

Hopefully someone can put me in the right direction.

BR

**Comment by mcgratta** [comment]
Make similar plots of heat release rate (HRR) and radiation heat release rate (RHRR).

**Reply by RNCFD** [reply]
Hi, they look quite similar:
![Picture2](https://github.com/user-attachments/assets/b3451748-f3d6-4928-84b6-fb52a87a7196)

**Comment by rmcdermo** [comment]
My guess is that this is related to thermo and fluid properties.  With C7 you have a heavier molecule, which affects density and hence buoyancy.  Further, the cp will be based on N2 for your FORMULA whereas C7 will be based on NASA polynomials.

**Reply by drjfloyd** [reply]
Your heptane reaction input is non-physical.  Non-physical inputs should be expected to lead to non-physical results.  Using the chemical formula for heptane with the heat of combustion of polyurethane is going to result in a non-physical value for EPUMO2.

**Reply by RNCFD** [reply]
I also think in this direction. The molucular weight is much heavier for Heptane (at least in the setup), 
The out files give:
![Picture3](https://github.com/user-attachments/assets/0727d187-e31a-46ef-a230-e80766d35ea0)

And further down:

![Picture4](https://github.com/user-attachments/assets/f85e9988-b5fa-4f86-ad73-02d2cd2bf77e)

I don't exactly know what the differences are and how FDS uses this. I will need more reading to better understand. Do you have a simple explanation for this?

I do agree that with unphysical setups you get unphysical results. The way I setup the reaction (heat of combustion and soot) are more for a polyurethane fire. The out files give these values:
![Picture5](https://github.com/user-attachments/assets/7a5938ca-3917-4b47-bb54-b4ed881c8da7)

So the EPUMO2 for heptane is much lower. You would expect higher (without my changes), do you think that this causes the lower temperatures and do you think that for the fire in which I usd the Heptane reaction these are too low?

**Reply by drjfloyd** [reply]
The real heat of combustion of hetpane is ~45 MJ / kg.  You have input 20 MJ/kg.  This means your EPUMO2 is half the actual value. Having such a low value is going to have an impact on the dynamics of the fire plume. Try running without the HEAT_OF_COMBUSTION input so you use the actual value for heptane and see what impact that has. The MW is probably also having some effect. You are  measuring temperature at a single point. That peak to ~1000 C is suggesting that at that time the measurement location is in the flame sheet. It would take much of a shift in that location to have a large impact on temperature.

**Reply by RNCFD** [reply]
Thank you, you defintely put me in the right direction. After some more reading I better understand the effect of changing the heat of combustion now. The values I got for EPUMO2 were not realistic anymore. I will setup a fuel/reaction which closer matches the values I will need to use.
